scan/context.go

```diff
@@ -112,19 +114,40 @@ func (s *Scanner) getContextFromReader(r io.Reader) (err error) {
 
 // getWithStatusError does an http.Get() and returns an error if the
 // status code is 4xx or 5xx.
+// It retries 10 times either if
```

Comment:
Comment can just be made generic: `it retries if either:`. Comments that are displaced from the variable name are likely to end up out of date